src/plugins/controls/public/plugin.ts

```diff
@@ -8,7 +8,18 @@
 
 import { CoreSetup, CoreStart, Plugin } from '@kbn/core/public';
 import { EmbeddableFactory } from '@kbn/embeddable-plugin/public';
+
+import {
+  ControlGroupContainerFactory,
+  CONTROL_GROUP_TYPE,
+  OPTIONS_LIST_CONTROL,
+  RANGE_SLIDER_CONTROL,
+  // TIME_SLIDER_CONTROL,
+} from '.';
+import { OptionsListEmbeddableFactory, OptionsListEmbeddableInput } from './options_list';
+import { RangeSliderEmbeddableFactory, RangeSliderEmbeddableInput } from './range_slider';
 import { pluginServices } from './services';
+import { controlsService } from './services/kibana/controls';
```

Comment:
We seem to be constantly going back and forth on how to organize imports, haha! Might be worth actually settling on something and adding it to our stylesheet so we can all be on the same page? Otherwise, our imports will just keep getting reorganized depending on who is editing the file 😆 

Not worth holding up this PR, though, obviously 👍

Reply:
Haha yeah we can discuss with Devon, but I prefer to order relative imports based on like folder traversal? Like here I started with `.` and ordered the nested dirs alphabetically bc IMO the parent dir should sit above the child dirs. It's kind of like a alphabetical depth first search type of ordering lol